utils/m0reportbug

```diff
@@ -699,6 +703,10 @@ sysrq_info() {
         [t]='show-task-states'
         [m]='show-memory-usage'
     )
+    if [[ "ro"==$(cat /proc/mounts | grep /proc/sysrq-trigger | awk '{print substr($4,1,2)}') ]] ; then
+        warn "/proc/sysrq-trigger is ro"
```

Comment:
Please add meaningful message here. e.g something similar to ". /proc/sysrq-trigger is read-only, won't be able to collect system information. "

Reply:
incorporated